DigitalMeLib/servers/gedis/GedisServer.py

```diff
@@ -84,41 +78,35 @@ def __init__(self, instance, data=None, parent=None, interactive=False, template
         if self.code_generated_dir not in sys.path:
             sys.path.append(self.code_generated_dir)
 
-        self.actors_add(namespace="system", path="%s/systemactors" % j.servers.gedis._dirpath)  # add the system actors
+        self.actors_add(namespace="system", path=j.sal.fs.joinPaths(
+            j.servers.gedis._dirpath, "systemactors"))  # add the system actors
 
-        self._sig_handler.append(gevent.signal(signal.SIGINT, self.stop))
+        for sig in [signal.SIGINT, signal.SIGTERM]:
+            self._sig_handler.append(gevent.signal(sig, self.stop))
 
-        # from gevent import monkey
-        # monkey.patch_thread() #TODO:*1 dirty hack, need to use gevent primitives, suggest to add flask server
-        # import threading
-        self.handler = Handler(self)
+        handler = Handler(self)
         if self.ssl:
             self.ssl_priv_key_path, self.ssl_cert_path = self.sslkeys_generate()
             # Server always supports SSL
             # client can use to talk to it in SSL or not
             self.redis_server = StreamServer(
                 (self.host, self.port),
                 spawn=Pool(),
-                handle=self.handler.handle_redis,
+                handle=handler.handle_redis,
                 keyfile=self.ssl_priv_key_path,
                 certfile=self.ssl_cert_path
             )
         else:
             self.redis_server = StreamServer(
                 (self.host, self.port),
                 spawn=Pool(),
-                handle=self.handler.handle_redis
+                handle=handler.handle_redis
             )
 
     # CODE GENERATION
-
     def code_generate_webclient(self):
```

Comment:
this is not needed too, now we are using open resty to directly send commands to gedis

Reply:
do you mean we don't support javascript client generation anymore at all ?

Reply:
yes

Reply:
for the record
from @andrewayoub :
> the point is that we don't even need this generated client to use gedis from js side
we are doing that now by directly call the cmds over websocket
and open resty convert these commands to tcp scoket
so we have a generic client